supersim_test.go

```diff
@@ -0,0 +1,64 @@
+package supersim
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func TestGenesisState(t *testing.T) {
+	logger := oplog.NewLogger(os.Stderr, oplog.DefaultCLIConfig())
+	supersim := NewSupersim(logger, &DefaultConfig)
+	supersim.Start(context.Background())
+	defer supersim.Stop(context.Background())
+
+	// Wait for anvil instances to start
+	time.Sleep(1000 * time.Millisecond)
```

Comment:
We should avoid adding sleeps as much as possible, this can result in tests sometimes failing and generally makes test suites take longer as they grow. `supersim` itself might need to return something to us or give us an event to listen to while we're waiting for everything to become healthy. Another approach is we can have a general `testutils.go` or something along those lines and we can have a waitFor function that takes a timeout and rpc url and we just poll one of the rpc endpoints until we get a response or the timeout is hit.

Reply:
good call, I added a `WaitForAnvilClientToBeReady` test util function